app/src/ui/lib/update-store.ts

```diff
@@ -155,22 +158,22 @@ class UpdateStore {
     }
   }
 
-  private getFeedURL(username: string): string {
-    return `${UpdatesURLBase}?version=${getVersion()}&username=${username}`
+  private getFeedURL(env: UpdateEnvironment): string {
+    return `${UpdatesURLBase}?version=${getVersion()}&env=${env}`
   }
 
   /**
-   * Check for updates using the given username.
+   * Check for updates.
    *
-   * @param username     - The username used to check for updates.
+   * @param env          - The environment to check for updates.
```

Comment:
Environment is such a loosely defined term. Could we document that we're talking about release channels here?